scripts/tso-brute.nse

```diff
@@ -94,32 +94,47 @@ Driver = {
 
     local commands = self.options['key1']
     local always_logon = self.options['key2']
+    local skip = self.options['skip']
     stdnse.debug(2,"Getting to TSO")
     local run = stdnse.strsplit(";%s*", commands)
+    stdnse.verbose(2,"Trying User ID/Password: %s/%s", user, pass)
```

Comment:
While debug does say which user id it is trying, especially at level 2, it contains a lot of extraneous information. Below you can the difference with just verbosity turned (`-v`) on vs debugging (`-d2`). I find for some longer brute force attempts/enumerations having a clean output makes it easier for me to monitor that status vs sifting through the debug information. 

Verbosity (-v)
```
nmap -p 23 -n -sV 10.10.0.201 --script tso-brute --script-args userdb=./nmap-dev-users.txt,passdb=nmap-dev-passdb.txt,brute.threads=1,brute.start=1,brute.useraspass=false,unpwdb.timelimit=0 -v

Starting Nmap 7.40SVN ( https://nmap.org ) at 2017-01-19 09:55 PST
NSE: Loaded 41 scripts for scanning.
NSE: Script Pre-scanning.
Initiating NSE at 09:55
Completed NSE at 09:55, 0.00s elapsed
Initiating NSE at 09:55
Completed NSE at 09:55, 0.00s elapsed
Initiating Ping Scan at 09:55
Scanning 10.10.0.201 [2 ports]
Completed Ping Scan at 09:55, 0.03s elapsed (1 total hosts)
Initiating Connect Scan at 09:55
Scanning 10.10.0.201 [1 port]
Discovered open port 23/tcp on 10.10.0.201
Completed Connect Scan at 09:55, 0.01s elapsed (1 total ports)
Initiating Service scan at 09:55
Scanning 1 service on 10.10.0.201
Completed Service scan at 09:55, 0.02s elapsed (1 service on 1 host)
NSE: Script scanning 10.10.0.201.
Initiating NSE at 09:55
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AHD/fake
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AGD/fake
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AHC/fake
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AGE/fake
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AIC/fake
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AGA/fake
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AHE/fake
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AIF/fake
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AGC/fake
Completed NSE at 09:55, 11.19s elapsed
Initiating NSE at 09:55
Completed NSE at 09:55, 0.00s elapsed
Nmap scan report for 10.10.0.201
Host is up (0.025s latency).
PORT   STATE SERVICE VERSION
23/tcp open  tn3270  IBM Telnet TN3270 (TN3270E)
| tso-brute:
|   Accounts: No valid accounts found
|_  Statistics: Performed 9 guesses in 9 seconds, average tps: 1.0

NSE: Script Post-scanning.
Initiating NSE at 09:55
Completed NSE at 09:55, 0.00s elapsed
Initiating NSE at 09:55
Completed NSE at 09:55, 0.00s elapsed
Read data files from: /usr/local/bin/../share/nmap
Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .
Nmap done: 1 IP address (1 host up) scanned in 11.68 seconds
```

Debug (-d2)

```
nmap -p 23 -n -sV 10.10.0.201 --script tso-brute --script-args userdb=./nmap-dev-users.txt,passdb=nmap-dev-passdb.txt,brute.threads=1,brute.start=1,brute.useraspass=false,unpwdb.timelimit=0 -d2

Starting Nmap 7.40SVN ( https://nmap.org ) at 2017-01-19 09:56 PST
Fetchfile found /usr/local/bin/../share/nmap/nmap-services
Fetchfile found /usr/local/bin/../share/nmap/nmap.xsl
The max # of sockets we are using is: 0
--------------- Timing report ---------------
  hostgroups: min 1, max 100000
  rtt-timeouts: init 1000, min 100, max 10000
  max-scan-delay: TCP 1000, UDP 1000, SCTP 1000
  parallelism: min 0, max 0
  max-retries: 10, host-timeout: 0
  min-rate: 0, max-rate: 0
---------------------------------------------
NSE: Using Lua 5.3.
Fetchfile found /usr/local/bin/../share/nmap/nse_main.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/lpeg-utility.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/stdnse.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/strict.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/script.db
NSE: Arguments from CLI: userdb=./nmap-dev-users.txt,passdb=nmap-dev-passdb.txt,brute.threads=1,brute.start=1,brute.useraspass=false,unpwdb.timelimit=0
NSE: Arguments parsed: userdb=./nmap-dev-users.txt,passdb=nmap-dev-passdb.txt,brute.threads=1,brute.start=1,brute.useraspass=false,unpwdb.timelimit=0
NSE: {
	["brute.threads"] = "1",
	["passdb"] = "nmap-dev-passdb.txt",
	["userdb"] = "./nmap-dev-users.txt",
	["brute.useraspass"] = "false",
	["unpwdb.timelimit"] = "0",
	["brute.start"] = "1",
}
Fetchfile found /usr/local/bin/../share/nmap/scripts/allseeingeye-info.nse
NSE: Script allseeingeye-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/comm.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/shortport.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/unittest.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/nsedebug.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/listop.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/bin.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/bit.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/amqp-info.nse
NSE: Script amqp-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/amqp.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/match.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/bacnet-info.nse
NSE: Script bacnet-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/unicode.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/cccam-version.nse
NSE: Script cccam-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/formulas.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/db2-das-info.nse
NSE: Script db2-das-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/docker-version.nse
NSE: Script docker-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/json.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/http.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/base64.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/sasl.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/smbauth.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/slaxml.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/url.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/drda-info.nse
NSE: Script drda-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/drda.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/enip-info.nse
NSE: Script enip-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/ipOps.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/fingerprint-strings.nse
NSE: Script fingerprint-strings.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/fox-info.nse
NSE: Script fox-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/freelancer-info.nse
NSE: Script freelancer-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/http-server-header.nse
NSE: Script http-server-header.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/iax2-version.nse
NSE: Script iax2-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/ike-version.nse
NSE: Script ike-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/ike.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/jdwp-version.nse
NSE: Script jdwp-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/maxdb-info.nse
NSE: Script maxdb-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/tab.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/strbuf.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/mcafee-epo-agent.nse
NSE: Script mcafee-epo-agent.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/mqtt-subscribe.nse
NSE: Script mqtt-subscribe.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/mqtt.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/murmur-version.nse
NSE: Script murmur-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/ndmp-version.nse
NSE: Script ndmp-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/ndmp.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/netbus-version.nse
NSE: Script netbus-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/omron-info.nse
NSE: Script omron-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/openlookup-info.nse
NSE: Script openlookup-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/oracle-tns-version.nse
NSE: Script oracle-tns-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/ovs-agent-version.nse
NSE: Script ovs-agent-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/pptp-version.nse
NSE: Script pptp-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/quake1-info.nse
NSE: Script quake1-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/quake3-info.nse
NSE: Script quake3-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/rfc868-time.nse
NSE: Script rfc868-time.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/datetime.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/rpc-grind.nse
NSE: Script rpc-grind.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/rpc.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/datafiles.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/rpcinfo.nse
NSE: Script rpcinfo.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/s7-info.nse
NSE: Script s7-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/skypev2-version.nse
NSE: Script skypev2-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/snmp-info.nse
NSE: Script snmp-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/snmp.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/asn1.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/creds.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/stun-version.nse
NSE: Script stun-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/stun.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/teamspeak2-version.nse
NSE: Script teamspeak2-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/tso-brute.nse
NSE: Script tso-brute.nse was selected by name.
Fetchfile found /usr/local/bin/../share/nmap/nselib/tn3270.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/brute.lua
Fetchfile found /usr/local/bin/../share/nmap/nselib/unpwdb.lua
Fetchfile found /usr/local/bin/../share/nmap/scripts/ventrilo-info.nse
NSE: Script ventrilo-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/wdb-version.nse
NSE: Script wdb-version.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/weblogic-t3-info.nse
NSE: Script weblogic-t3-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/scripts/xmpp-info.nse
NSE: Script xmpp-info.nse was selected by category.
Fetchfile found /usr/local/bin/../share/nmap/nselib/xmpp.lua
NSE: Loaded 41 scripts for scanning.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/allseeingeye-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/amqp-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/bacnet-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/cccam-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/db2-das-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/docker-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/drda-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/enip-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/fingerprint-strings.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/fox-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/freelancer-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/http-server-header.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/iax2-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/ike-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/jdwp-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/maxdb-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/mcafee-epo-agent.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/mqtt-subscribe.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/murmur-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/ndmp-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/netbus-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/omron-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/openlookup-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/oracle-tns-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/ovs-agent-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/pptp-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/quake1-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/quake3-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/rfc868-time.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/rpc-grind.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/rpcinfo.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/s7-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/skypev2-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/snmp-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/stun-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/teamspeak2-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/tso-brute.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/ventrilo-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/wdb-version.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/weblogic-t3-info.nse'.
NSE: Loaded '/usr/local/bin/../share/nmap/scripts/xmpp-info.nse'.
NSE: Script Pre-scanning.
NSE: Starting runlevel 1 (of 2) scan.
Initiating NSE at 09:56
Completed NSE at 09:56, 0.00s elapsed
NSE: Starting runlevel 2 (of 2) scan.
Initiating NSE at 09:56
Completed NSE at 09:56, 0.00s elapsed
Fetchfile found /usr/local/bin/../share/nmap/nmap-payloads
Initiating Ping Scan at 09:56
Scanning 10.10.0.201 [2 ports]
ultrascan_host_probe_update called for machine 10.10.0.201 state UNKNOWN -> HOST_UP (trynum 0 time: 6740)
Changing ping technique for 10.10.0.201 to connect to port 80
Changing global ping host to 10.10.0.201.
Completed Ping Scan at 09:56, 0.01s elapsed (1 total hosts)
Overall sending rates: 293.17 packets / s.
Initiating Connect Scan at 09:56
Scanning 10.10.0.201 [1 port]
Discovered open port 23/tcp on 10.10.0.201
Changing global ping host to 10.10.0.201.
Completed Connect Scan at 09:56, 0.01s elapsed (1 total ports)
Overall sending rates: 65.87 packets / s.
Fetchfile found /usr/local/bin/../share/nmap/nmap-service-probes
Initiating Service scan at 09:56
Scanning 1 service on 10.10.0.201
NSOCK INFO [0.4530s] nsock_iod_new2(): nsock_iod_new (IOD #1)
Starting probes against new service: 10.10.0.201:23 (tcp)
NSOCK INFO [0.4530s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #1) EID 8
NSOCK INFO [0.4640s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 8 [10.10.0.201:23]
Service scan sending probe NULL to 10.10.0.201:23 (tcp)
NSOCK INFO [0.4640s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 6000ms) EID 18
NSOCK INFO [0.4750s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 18 [10.10.0.201:23] (3 bytes): ..(
Service scan match (Probe NULL matched with NULL line 4687): 10.10.0.201:23 is tn3270.  Version: |IBM Telnet TN3270||TN3270E|
NSOCK INFO [0.4750s] nsock_iod_delete(): nsock_iod_delete (IOD #1)
Completed Service scan at 09:56, 0.02s elapsed (1 service on 1 host)
NSE: Script scanning 10.10.0.201.
NSE: Starting runlevel 1 (of 2) scan.
Initiating NSE at 09:56
NSE: Starting tso-brute M:2d918f8 against 10.10.0.201:23.
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Checking for TSO
NSOCK INFO [0.4760s] nsock_iod_new2(): nsock_iod_new (IOD #1)
NSOCK INFO [0.4790s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #1) EID 8
NSOCK INFO [0.4850s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 8 [10.10.0.201:23]
NSOCK INFO [0.4850s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 7000ms) EID 18
NSOCK INFO [0.4920s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 18 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [0.4930s] nsock_write(): Write request for 3 bytes to IOD #1 EID 27 [10.10.0.201:23]
NSOCK INFO [0.4930s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 27 [10.10.0.201:23]
NSOCK INFO [0.4930s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 7000ms) EID 34
NSOCK INFO [0.4970s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 34 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [0.4970s] nsock_write(): Write request for 3 bytes to IOD #1 EID 43 [10.10.0.201:23]
NSOCK INFO [0.4970s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 43 [10.10.0.201:23]
NSOCK INFO [0.4970s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 7000ms) EID 50
NSOCK INFO [0.5010s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 50 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [0.5010s] nsock_write(): Write request for 16 bytes to IOD #1 EID 59 [10.10.0.201:23]
NSOCK INFO [0.5010s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 59 [10.10.0.201:23]
NSOCK INFO [0.5010s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 7000ms) EID 66
NSOCK INFO [0.5130s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 66 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [0.5140s] nsock_write(): Write request for 3 bytes to IOD #1 EID 75 [10.10.0.201:23]
NSOCK INFO [0.5140s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 75 [10.10.0.201:23]
NSOCK INFO [0.5140s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 7000ms) EID 82
NSOCK INFO [0.5150s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 82 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [0.5160s] nsock_write(): Write request for 3 bytes to IOD #1 EID 91 [10.10.0.201:23]
NSOCK INFO [0.5160s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 91 [10.10.0.201:23]
NSOCK INFO [0.5160s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 7000ms) EID 98
NSOCK INFO [0.5590s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 98 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [0.5590s] nsock_write(): Write request for 3 bytes to IOD #1 EID 107 [10.10.0.201:23]
NSOCK INFO [0.5590s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 107 [10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [0.5590s] nsock_write(): Write request for 3 bytes to IOD #1 EID 115 [10.10.0.201:23]
NSOCK INFO [0.5590s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 115 [10.10.0.201:23]
NSOCK INFO [0.5600s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 7000ms) EID 122
NSOCK INFO [0.8300s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 122 [10.10.0.201:23] (524 bytes)
NSOCK INFO [0.8310s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 7000ms) EID 130
NSOCK INFO [0.8310s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 130 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute M:2d918f8 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]                                                                                W
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute M:2d918f8 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute M:2d918f8 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute M:2d918f8 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute M:2d918f8 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute M:2d918f8 10.10.0.201:23] fficials.
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0x7d50d41150d1a3a296ffef
NSOCK INFO [0.8410s] nsock_write(): Write request for 11 bytes to IOD #1 EID 139 [10.10.0.201:23]
NSOCK INFO [0.8410s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 139 [10.10.0.201:23]
NSOCK INFO [0.8410s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 200ms) EID 146
NSOCK INFO [0.8960s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 146 [10.10.0.201:23] (524 bytes)
NSOCK INFO [0.8970s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 200ms) EID 154
NSOCK INFO [0.8970s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 154 [10.10.0.201:23] (122 bytes)
NSOCK INFO [0.9070s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 200ms) EID 162
NSOCK INFO [0.9640s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 162 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [0.9650s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 200ms) EID 170
NSOCK INFO [0.9810s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 170 [10.10.0.201:23] (44 bytes): ...]..@.@@...........@.....@......@`.@..P...
NSOCK INFO [0.9820s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 200ms) EID 178
NSOCK INFO [0.9850s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 178 [10.10.0.201:23] (4 bytes): ....
NSOCK INFO [0.9850s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 200ms) EID 186
NSOCK INFO [1.1850s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 186 [10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute M:2d918f8 10.10.0.201:23] IKJ56700A ENTER USERID -
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0x7dc1d711c1509596a399858193ffef
NSOCK INFO [1.1970s] nsock_write(): Write request for 15 bytes to IOD #1 EID 195 [10.10.0.201:23]
NSOCK INFO [1.1970s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 195 [10.10.0.201:23]
NSOCK INFO [1.1970s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 200ms) EID 202
NSOCK INFO [1.2760s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 202 [10.10.0.201:23] (524 bytes)
NSOCK INFO [1.2780s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 200ms) EID 210
NSOCK INFO [1.2780s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 210 [10.10.0.201:23] (582 bytes)
NSOCK INFO [1.2940s] nsock_read(): Read request from IOD #1 [10.10.0.201:23] (timeout: 200ms) EID 218
NSOCK INFO [1.4940s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 218 [10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xf3c6e3ffef
NSOCK INFO [1.4980s] nsock_write(): Write request for 5 bytes to IOD #1 EID 227 [10.10.0.201:23]
NSOCK INFO [1.4980s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 227 [10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Disconnecting
NSOCK INFO [1.4980s] nsock_iod_delete(): nsock_iod_delete (IOD #1)
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Starting TSO Brute Force
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Checking for IKJ56700A message skip
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Connecting TN3270 to 10.10.0.201:23
NSOCK INFO [1.4980s] nsock_iod_new2(): nsock_iod_new (IOD #2)
NSOCK INFO [1.4980s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #2) EID 232
NSOCK INFO [1.5040s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 232 [10.10.0.201:23]
NSOCK INFO [1.5040s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 7000ms) EID 242
NSOCK INFO [1.5180s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 242 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [1.5180s] nsock_write(): Write request for 3 bytes to IOD #2 EID 251 [10.10.0.201:23]
NSOCK INFO [1.5180s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 251 [10.10.0.201:23]
NSOCK INFO [1.5180s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 7000ms) EID 258
NSOCK INFO [1.5310s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 258 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [1.5310s] nsock_write(): Write request for 3 bytes to IOD #2 EID 267 [10.10.0.201:23]
NSOCK INFO [1.5310s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 267 [10.10.0.201:23]
NSOCK INFO [1.5310s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 7000ms) EID 274
NSOCK INFO [1.5350s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 274 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [1.5350s] nsock_write(): Write request for 16 bytes to IOD #2 EID 283 [10.10.0.201:23]
NSOCK INFO [1.5350s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 283 [10.10.0.201:23]
NSOCK INFO [1.5350s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 7000ms) EID 290
NSOCK INFO [1.5460s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 290 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [1.5460s] nsock_write(): Write request for 3 bytes to IOD #2 EID 299 [10.10.0.201:23]
NSOCK INFO [1.5460s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 299 [10.10.0.201:23]
NSOCK INFO [1.5460s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 7000ms) EID 306
NSOCK INFO [1.5470s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 306 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [1.5480s] nsock_write(): Write request for 3 bytes to IOD #2 EID 315 [10.10.0.201:23]
NSOCK INFO [1.5480s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 315 [10.10.0.201:23]
NSOCK INFO [1.5480s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 7000ms) EID 322
NSOCK INFO [1.5930s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 322 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [1.5930s] nsock_write(): Write request for 3 bytes to IOD #2 EID 331 [10.10.0.201:23]
NSOCK INFO [1.5930s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 331 [10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [1.5930s] nsock_write(): Write request for 3 bytes to IOD #2 EID 339 [10.10.0.201:23]
NSOCK INFO [1.5930s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 339 [10.10.0.201:23]
NSOCK INFO [1.5940s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 7000ms) EID 346
NSOCK INFO [1.8670s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 346 [10.10.0.201:23] (524 bytes)
NSOCK INFO [1.8680s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 7000ms) EID 354
NSOCK INFO [1.8680s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 354 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Displaying initial TN3270 Screen:
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute M:2d918f8 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]                                                                                W
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute M:2d918f8 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute M:2d918f8 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute M:2d918f8 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute M:2d918f8 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute M:2d918f8 10.10.0.201:23] fficials.
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Not using LOGON command, testing adding userid to command
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending tso FAKEUSER
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Sending data: 0x7d505d1150d1a3a29640c6c1d2c5e4e2c5d9ffef
NSOCK INFO [1.8800s] nsock_write(): Write request for 20 bytes to IOD #2 EID 363 [10.10.0.201:23]
NSOCK INFO [1.8800s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 363 [10.10.0.201:23]
NSOCK INFO [1.8800s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 200ms) EID 370
NSOCK INFO [1.9140s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 370 [10.10.0.201:23] (524 bytes)
NSOCK INFO [1.9150s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 200ms) EID 378
NSOCK INFO [1.9150s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 378 [10.10.0.201:23] (122 bytes)
NSOCK INFO [1.9250s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 200ms) EID 386
NSOCK INFO [1.9870s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 386 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [1.9870s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 200ms) EID 394
NSOCK INFO [2.0000s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 394 [10.10.0.201:23] (93 bytes)
NSOCK INFO [2.0020s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 200ms) EID 402
NSOCK INFO [2.0040s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 402 [10.10.0.201:23] (4 bytes): ....
NSOCK INFO [2.0040s] nsock_read(): Read request from IOD #2 [10.10.0.201:23] (timeout: 200ms) EID 410
NSOCK INFO [2.2050s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 410 [10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute M:2d918f8 10.10.0.201:23] IKJ56710I INVALID USERID, FAKEUSER
NSE: [tso-brute M:2d918f8 10.10.0.201:23] IKJ56703A REENTER THIS OPERAND -
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23]
NSE: [tso-brute M:2d918f8 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute M:2d918f8 10.10.0.201:23] IKJ56700A message skip supported
NSE: tso-brute M:2d918f8 spawning new thread (thread: 0x1f65128).
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Status: #threads = 1, #retry_accounts = 0, initial_accounts_exhausted = false, waiting = 0
NSOCK INFO [2.2100s] nsock_iod_new2(): nsock_iod_new (IOD #3)
NSOCK INFO [2.2100s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #3) EID 416
NSOCK INFO [2.2180s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 416 [10.10.0.201:23]
NSOCK INFO [2.2180s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 7000ms) EID 426
NSOCK INFO [2.2290s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 426 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [2.2290s] nsock_write(): Write request for 3 bytes to IOD #3 EID 435 [10.10.0.201:23]
NSOCK INFO [2.2290s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 435 [10.10.0.201:23]
NSOCK INFO [2.2300s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 7000ms) EID 442
NSOCK INFO [2.2350s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 442 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [2.2360s] nsock_write(): Write request for 3 bytes to IOD #3 EID 451 [10.10.0.201:23]
NSOCK INFO [2.2360s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 451 [10.10.0.201:23]
NSOCK INFO [2.2360s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 7000ms) EID 458
NSOCK INFO [2.2410s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 458 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [2.2410s] nsock_write(): Write request for 16 bytes to IOD #3 EID 467 [10.10.0.201:23]
NSOCK INFO [2.2410s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 467 [10.10.0.201:23]
NSOCK INFO [2.2410s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 7000ms) EID 474
NSOCK INFO [2.2480s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 474 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [2.2480s] nsock_write(): Write request for 3 bytes to IOD #3 EID 483 [10.10.0.201:23]
NSOCK INFO [2.2480s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 483 [10.10.0.201:23]
NSOCK INFO [2.2480s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 7000ms) EID 490
NSOCK INFO [2.2540s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 490 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [2.2540s] nsock_write(): Write request for 3 bytes to IOD #3 EID 499 [10.10.0.201:23]
NSOCK INFO [2.2540s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 499 [10.10.0.201:23]
NSOCK INFO [2.2540s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 7000ms) EID 506
NSOCK INFO [2.2950s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 506 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [2.2950s] nsock_write(): Write request for 3 bytes to IOD #3 EID 515 [10.10.0.201:23]
NSOCK INFO [2.2950s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 515 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [2.2950s] nsock_write(): Write request for 3 bytes to IOD #3 EID 523 [10.10.0.201:23]
NSOCK INFO [2.2950s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 523 [10.10.0.201:23]
NSOCK INFO [2.2960s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 7000ms) EID 530
NSOCK INFO [2.5590s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 530 [10.10.0.201:23] (524 bytes)
NSOCK INFO [2.5600s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 7000ms) EID 538
NSOCK INFO [2.5600s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 538 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]                                                                                W
NSE: [tso-brute W:1f65128 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute W:1f65128 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute W:1f65128 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute W:1f65128 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute W:1f65128 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute W:1f65128 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute W:1f65128 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute W:1f65128 10.10.0.201:23] fficials.
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying @AHD/fake against 10.10.0.201:23
NSE: [tso-brute W:1f65128 10.10.0.201:23] Getting to TSO
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AHD/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying User ID: @AHD
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7d50d91150d1a3a296407cc1c8c4ffef
NSOCK INFO [2.5720s] nsock_write(): Write request for 16 bytes to IOD #3 EID 547 [10.10.0.201:23]
NSOCK INFO [2.5720s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 547 [10.10.0.201:23]
NSOCK INFO [2.5730s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 554
NSOCK INFO [2.6080s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 554 [10.10.0.201:23] (524 bytes)
NSOCK INFO [2.6090s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 562
NSOCK INFO [2.6090s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 562 [10.10.0.201:23] (122 bytes)
NSOCK INFO [2.6190s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 570
NSOCK INFO [2.6660s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 570 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [2.6660s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 578
NSOCK INFO [2.6870s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 578 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [2.6880s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 586
NSOCK INFO [2.6900s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 586 [10.10.0.201:23] (524 bytes)
NSOCK INFO [2.6910s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 594
NSOCK INFO [2.6910s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 594 [10.10.0.201:23] (582 bytes)
NSOCK INFO [2.6910s] nsock_iod_delete(): nsock_iod_delete (IOD #2)
NSOCK INFO [2.7070s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 602
NSOCK INFO [2.9070s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 602 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User ID: @AHD
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AHD                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] @AHD is a valid TSO User ID. Trying Password: fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7dc9c711c9c386819285ffef
NSOCK INFO [2.9200s] nsock_write(): Write request for 12 bytes to IOD #3 EID 611 [10.10.0.201:23]
NSOCK INFO [2.9200s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 611 [10.10.0.201:23]
NSOCK INFO [2.9200s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 618
NSOCK INFO [2.9340s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 618 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [2.9340s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 626
NSOCK INFO [2.9380s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 626 [10.10.0.201:23] (8 bytes): ...@@...
NSOCK INFO [2.9390s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 634
NSOCK INFO [2.9630s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 634 [10.10.0.201:23] (524 bytes)
NSOCK INFO [2.9650s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 642
NSOCK INFO [2.9650s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 642 [10.10.0.201:23] (524 bytes)
NSOCK INFO [2.9670s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 650
NSOCK INFO [2.9670s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 650 [10.10.0.201:23] (58 bytes): @@@@@@@@@@@@......|@........@@@@@~~~n....|@@@@@@@@........
NSOCK INFO [2.9800s] nsock_read(): Read request from IOD #3 [10.10.0.201:23] (timeout: 200ms) EID 658
NSOCK INFO [3.1800s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 658 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User/Pass: @AHD/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56421I PASSWORD NOT AUTHORIZED FOR USERID
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56429A REENTER -
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AHD                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Disconnecting
NSOCK INFO [3.1800s] nsock_iod_delete(): nsock_iod_delete (IOD #3)
NSOCK INFO [3.1800s] nsock_iod_new2(): nsock_iod_new (IOD #4)
NSOCK INFO [3.1900s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #4) EID 664
NSOCK INFO [3.1980s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 664 [10.10.0.201:23]
NSOCK INFO [3.1980s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 7000ms) EID 674
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Status: #threads = 1, #retry_accounts = 0, initial_accounts_exhausted = false, waiting = 0
NSOCK INFO [3.2100s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 674 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [3.2100s] nsock_write(): Write request for 3 bytes to IOD #4 EID 683 [10.10.0.201:23]
NSOCK INFO [3.2100s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 683 [10.10.0.201:23]
NSOCK INFO [3.2100s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 7000ms) EID 690
NSOCK INFO [3.2200s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 690 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [3.2200s] nsock_write(): Write request for 3 bytes to IOD #4 EID 699 [10.10.0.201:23]
NSOCK INFO [3.2200s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 699 [10.10.0.201:23]
NSOCK INFO [3.2200s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 7000ms) EID 706
NSOCK INFO [3.2280s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 706 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [3.2280s] nsock_write(): Write request for 16 bytes to IOD #4 EID 715 [10.10.0.201:23]
NSOCK INFO [3.2280s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 715 [10.10.0.201:23]
NSOCK INFO [3.2280s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 7000ms) EID 722
NSOCK INFO [3.2340s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 722 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [3.2350s] nsock_write(): Write request for 3 bytes to IOD #4 EID 731 [10.10.0.201:23]
NSOCK INFO [3.2350s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 731 [10.10.0.201:23]
NSOCK INFO [3.2350s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 7000ms) EID 738
NSOCK INFO [3.2370s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 738 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [3.2380s] nsock_write(): Write request for 3 bytes to IOD #4 EID 747 [10.10.0.201:23]
NSOCK INFO [3.2380s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 747 [10.10.0.201:23]
NSOCK INFO [3.2380s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 7000ms) EID 754
NSOCK INFO [3.2790s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 754 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [3.2790s] nsock_write(): Write request for 3 bytes to IOD #4 EID 763 [10.10.0.201:23]
NSOCK INFO [3.2790s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 763 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [3.2800s] nsock_write(): Write request for 3 bytes to IOD #4 EID 771 [10.10.0.201:23]
NSOCK INFO [3.2800s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 771 [10.10.0.201:23]
NSOCK INFO [3.2800s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 7000ms) EID 778
NSOCK INFO [3.5200s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 778 [10.10.0.201:23] (524 bytes)
NSOCK INFO [3.5220s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 7000ms) EID 786
NSOCK INFO [3.5220s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 786 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]                                                                                W
NSE: [tso-brute W:1f65128 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute W:1f65128 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute W:1f65128 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute W:1f65128 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute W:1f65128 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute W:1f65128 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute W:1f65128 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute W:1f65128 10.10.0.201:23] fficials.
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying @AGD/fake against 10.10.0.201:23
NSE: [tso-brute W:1f65128 10.10.0.201:23] Getting to TSO
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AGD/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying User ID: @AGD
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7d50d91150d1a3a296407cc1c7c4ffef
NSOCK INFO [3.5330s] nsock_write(): Write request for 16 bytes to IOD #4 EID 795 [10.10.0.201:23]
NSOCK INFO [3.5330s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 795 [10.10.0.201:23]
NSOCK INFO [3.5330s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 802
NSOCK INFO [3.5570s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 802 [10.10.0.201:23] (524 bytes)
NSOCK INFO [3.5590s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 810
NSOCK INFO [3.5590s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 810 [10.10.0.201:23] (122 bytes)
NSOCK INFO [3.5670s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 818
NSOCK INFO [3.5990s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 818 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [3.5990s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 826
NSOCK INFO [3.6200s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 826 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [3.6210s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 834
NSOCK INFO [3.6230s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 834 [10.10.0.201:23] (524 bytes)
NSOCK INFO [3.6250s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 842
NSOCK INFO [3.6250s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 842 [10.10.0.201:23] (582 bytes)
NSOCK INFO [3.6390s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 850
NSOCK INFO [3.8390s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 850 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User ID: @AGD
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AGD                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] @AGD is a valid TSO User ID. Trying Password: fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7dc9c711c9c386819285ffef
NSOCK INFO [3.8530s] nsock_write(): Write request for 12 bytes to IOD #4 EID 859 [10.10.0.201:23]
NSOCK INFO [3.8530s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 859 [10.10.0.201:23]
NSOCK INFO [3.8530s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 866
NSOCK INFO [3.8720s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 866 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [3.8720s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 874
NSOCK INFO [3.8790s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 874 [10.10.0.201:23] (8 bytes): ...@@...
NSOCK INFO [3.8800s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 882
NSOCK INFO [3.9220s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 882 [10.10.0.201:23] (524 bytes)
NSOCK INFO [3.9240s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 890
NSOCK INFO [3.9240s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 890 [10.10.0.201:23] (582 bytes)
NSOCK INFO [3.9380s] nsock_read(): Read request from IOD #4 [10.10.0.201:23] (timeout: 200ms) EID 898
NSOCK INFO [4.1380s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 898 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User/Pass: @AGD/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56421I PASSWORD NOT AUTHORIZED FOR USERID
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56429A REENTER -
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AGD                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Disconnecting
NSOCK INFO [4.1380s] nsock_iod_delete(): nsock_iod_delete (IOD #4)
NSOCK INFO [4.1380s] nsock_iod_new2(): nsock_iod_new (IOD #5)
NSOCK INFO [4.1500s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #5) EID 904
NSOCK INFO [4.1550s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 904 [10.10.0.201:23]
NSOCK INFO [4.1550s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 7000ms) EID 914
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Status: #threads = 1, #retry_accounts = 0, initial_accounts_exhausted = false, waiting = 0
NSOCK INFO [4.1700s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 914 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [4.1700s] nsock_write(): Write request for 3 bytes to IOD #5 EID 923 [10.10.0.201:23]
NSOCK INFO [4.1700s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 923 [10.10.0.201:23]
NSOCK INFO [4.1700s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 7000ms) EID 930
NSOCK INFO [4.1760s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 930 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [4.1760s] nsock_write(): Write request for 3 bytes to IOD #5 EID 939 [10.10.0.201:23]
NSOCK INFO [4.1760s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 939 [10.10.0.201:23]
NSOCK INFO [4.1760s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 7000ms) EID 946
NSOCK INFO [4.1810s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 946 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [4.1810s] nsock_write(): Write request for 16 bytes to IOD #5 EID 955 [10.10.0.201:23]
NSOCK INFO [4.1810s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 955 [10.10.0.201:23]
NSOCK INFO [4.1810s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 7000ms) EID 962
NSOCK INFO [4.1870s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 962 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [4.1870s] nsock_write(): Write request for 3 bytes to IOD #5 EID 971 [10.10.0.201:23]
NSOCK INFO [4.1870s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 971 [10.10.0.201:23]
NSOCK INFO [4.1870s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 7000ms) EID 978
NSOCK INFO [4.1890s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 978 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [4.1890s] nsock_write(): Write request for 3 bytes to IOD #5 EID 987 [10.10.0.201:23]
NSOCK INFO [4.1890s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 987 [10.10.0.201:23]
NSOCK INFO [4.1890s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 7000ms) EID 994
NSOCK INFO [4.2300s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 994 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [4.2300s] nsock_write(): Write request for 3 bytes to IOD #5 EID 1003 [10.10.0.201:23]
NSOCK INFO [4.2300s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1003 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [4.2300s] nsock_write(): Write request for 3 bytes to IOD #5 EID 1011 [10.10.0.201:23]
NSOCK INFO [4.2300s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1011 [10.10.0.201:23]
NSOCK INFO [4.2310s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 7000ms) EID 1018
NSOCK INFO [4.4870s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1018 [10.10.0.201:23] (524 bytes)
NSOCK INFO [4.4880s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 7000ms) EID 1026
NSOCK INFO [4.4880s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1026 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]                                                                                W
NSE: [tso-brute W:1f65128 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute W:1f65128 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute W:1f65128 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute W:1f65128 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute W:1f65128 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute W:1f65128 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute W:1f65128 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute W:1f65128 10.10.0.201:23] fficials.
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying @AHC/fake against 10.10.0.201:23
NSE: [tso-brute W:1f65128 10.10.0.201:23] Getting to TSO
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AHC/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying User ID: @AHC
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7d50d91150d1a3a296407cc1c8c3ffef
NSOCK INFO [4.4990s] nsock_write(): Write request for 16 bytes to IOD #5 EID 1035 [10.10.0.201:23]
NSOCK INFO [4.4990s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1035 [10.10.0.201:23]
NSOCK INFO [4.4990s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1042
NSOCK INFO [4.5310s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1042 [10.10.0.201:23] (524 bytes)
NSOCK INFO [4.5320s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1050
NSOCK INFO [4.5320s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1050 [10.10.0.201:23] (122 bytes)
NSOCK INFO [4.5410s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1058
NSOCK INFO [4.5810s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1058 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [4.5820s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1066
NSOCK INFO [4.6030s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1066 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [4.6030s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1074
NSOCK INFO [4.6060s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1074 [10.10.0.201:23] (524 bytes)
NSOCK INFO [4.6080s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1082
NSOCK INFO [4.6080s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1082 [10.10.0.201:23] (582 bytes)
NSOCK INFO [4.6230s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1090
NSOCK INFO [4.8230s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 1090 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User ID: @AHC
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AHC                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] @AHC is a valid TSO User ID. Trying Password: fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7dc9c711c9c386819285ffef
NSOCK INFO [4.8370s] nsock_write(): Write request for 12 bytes to IOD #5 EID 1099 [10.10.0.201:23]
NSOCK INFO [4.8370s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1099 [10.10.0.201:23]
NSOCK INFO [4.8370s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1106
NSOCK INFO [4.8560s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1106 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [4.8570s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1114
NSOCK INFO [4.8620s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1114 [10.10.0.201:23] (8 bytes): ...@@...
NSOCK INFO [4.8630s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1122
NSOCK INFO [4.9010s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1122 [10.10.0.201:23] (524 bytes)
NSOCK INFO [4.9020s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1130
NSOCK INFO [4.9020s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1130 [10.10.0.201:23] (524 bytes)
NSOCK INFO [4.9040s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1138
NSOCK INFO [4.9040s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1138 [10.10.0.201:23] (58 bytes): @@@@@@@@@@@@......|@........@@@@@~~~n....|@@@@@@@@........
NSOCK INFO [4.9160s] nsock_read(): Read request from IOD #5 [10.10.0.201:23] (timeout: 200ms) EID 1146
NSOCK INFO [5.1160s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 1146 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User/Pass: @AHC/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56421I PASSWORD NOT AUTHORIZED FOR USERID
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56429A REENTER -
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AHC                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Disconnecting
NSOCK INFO [5.1160s] nsock_iod_delete(): nsock_iod_delete (IOD #5)
NSOCK INFO [5.1160s] nsock_iod_new2(): nsock_iod_new (IOD #6)
NSOCK INFO [5.1270s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #6) EID 1152
NSOCK INFO [5.1340s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 1152 [10.10.0.201:23]
NSOCK INFO [5.1340s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 7000ms) EID 1162
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Status: #threads = 1, #retry_accounts = 0, initial_accounts_exhausted = false, waiting = 0
NSOCK INFO [5.1520s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1162 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [5.1520s] nsock_write(): Write request for 3 bytes to IOD #6 EID 1171 [10.10.0.201:23]
NSOCK INFO [5.1520s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1171 [10.10.0.201:23]
NSOCK INFO [5.1520s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 7000ms) EID 1178
NSOCK INFO [5.1610s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1178 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [5.1620s] nsock_write(): Write request for 3 bytes to IOD #6 EID 1187 [10.10.0.201:23]
NSOCK INFO [5.1620s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1187 [10.10.0.201:23]
NSOCK INFO [5.1620s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 7000ms) EID 1194
NSOCK INFO [5.1690s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1194 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [5.1690s] nsock_write(): Write request for 16 bytes to IOD #6 EID 1203 [10.10.0.201:23]
NSOCK INFO [5.1690s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1203 [10.10.0.201:23]
NSOCK INFO [5.1690s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 7000ms) EID 1210
NSOCK INFO [5.1740s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1210 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [5.1740s] nsock_write(): Write request for 3 bytes to IOD #6 EID 1219 [10.10.0.201:23]
NSOCK INFO [5.1740s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1219 [10.10.0.201:23]
NSOCK INFO [5.1740s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 7000ms) EID 1226
NSOCK INFO [5.1760s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1226 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [5.1770s] nsock_write(): Write request for 3 bytes to IOD #6 EID 1235 [10.10.0.201:23]
NSOCK INFO [5.1770s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1235 [10.10.0.201:23]
NSOCK INFO [5.1770s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 7000ms) EID 1242
NSOCK INFO [5.2200s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1242 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [5.2200s] nsock_write(): Write request for 3 bytes to IOD #6 EID 1251 [10.10.0.201:23]
NSOCK INFO [5.2200s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1251 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [5.2200s] nsock_write(): Write request for 3 bytes to IOD #6 EID 1259 [10.10.0.201:23]
NSOCK INFO [5.2200s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1259 [10.10.0.201:23]
NSOCK INFO [5.2210s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 7000ms) EID 1266
NSOCK INFO [5.4810s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1266 [10.10.0.201:23] (524 bytes)
NSOCK INFO [5.4830s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 7000ms) EID 1274
NSOCK INFO [5.4830s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1274 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]                                                                                W
NSE: [tso-brute W:1f65128 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute W:1f65128 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute W:1f65128 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute W:1f65128 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute W:1f65128 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute W:1f65128 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute W:1f65128 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute W:1f65128 10.10.0.201:23] fficials.
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying @AGE/fake against 10.10.0.201:23
NSE: [tso-brute W:1f65128 10.10.0.201:23] Getting to TSO
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AGE/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying User ID: @AGE
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7d50d91150d1a3a296407cc1c7c5ffef
NSOCK INFO [5.4940s] nsock_write(): Write request for 16 bytes to IOD #6 EID 1283 [10.10.0.201:23]
NSOCK INFO [5.4940s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1283 [10.10.0.201:23]
NSOCK INFO [5.4940s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1290
NSOCK INFO [5.5580s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1290 [10.10.0.201:23] (524 bytes)
NSOCK INFO [5.5600s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1298
NSOCK INFO [5.5600s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1298 [10.10.0.201:23] (122 bytes)
NSOCK INFO [5.5680s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1306
NSOCK INFO [5.6500s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1306 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [5.6510s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1314
NSOCK INFO [5.6790s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1314 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [5.6800s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1322
NSOCK INFO [5.6820s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1322 [10.10.0.201:23] (524 bytes)
NSOCK INFO [5.6830s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1330
NSOCK INFO [5.6830s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1330 [10.10.0.201:23] (582 bytes)
NSOCK INFO [5.6990s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1338
NSOCK INFO [5.8990s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 1338 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User ID: @AGE
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AGE                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] @AGE is a valid TSO User ID. Trying Password: fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7dc9c711c9c386819285ffef
NSOCK INFO [5.9140s] nsock_write(): Write request for 12 bytes to IOD #6 EID 1347 [10.10.0.201:23]
NSOCK INFO [5.9140s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1347 [10.10.0.201:23]
NSOCK INFO [5.9140s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1354
NSOCK INFO [5.9260s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1354 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [5.9270s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1362
NSOCK INFO [5.9370s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1362 [10.10.0.201:23] (8 bytes): ...@@...
NSOCK INFO [5.9370s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1370
NSOCK INFO [5.9600s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1370 [10.10.0.201:23] (524 bytes)
NSOCK INFO [5.9620s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1378
NSOCK INFO [5.9620s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1378 [10.10.0.201:23] (582 bytes)
NSOCK INFO [5.9760s] nsock_read(): Read request from IOD #6 [10.10.0.201:23] (timeout: 200ms) EID 1386
NSOCK INFO [6.1760s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 1386 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User/Pass: @AGE/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56421I PASSWORD NOT AUTHORIZED FOR USERID
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56429A REENTER -
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AGE                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Disconnecting
NSOCK INFO [6.1760s] nsock_iod_delete(): nsock_iod_delete (IOD #6)
NSOCK INFO [6.1760s] nsock_iod_new2(): nsock_iod_new (IOD #7)
NSOCK INFO [6.1890s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #7) EID 1392
NSOCK INFO [6.1950s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 1392 [10.10.0.201:23]
NSOCK INFO [6.1950s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 7000ms) EID 1402
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Status: #threads = 1, #retry_accounts = 0, initial_accounts_exhausted = false, waiting = 0
NSOCK INFO [6.2110s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1402 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [6.2110s] nsock_write(): Write request for 3 bytes to IOD #7 EID 1411 [10.10.0.201:23]
NSOCK INFO [6.2110s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1411 [10.10.0.201:23]
NSOCK INFO [6.2110s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 7000ms) EID 1418
NSOCK INFO [6.2240s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1418 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [6.2240s] nsock_write(): Write request for 3 bytes to IOD #7 EID 1427 [10.10.0.201:23]
NSOCK INFO [6.2240s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1427 [10.10.0.201:23]
NSOCK INFO [6.2240s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 7000ms) EID 1434
NSOCK INFO [6.2320s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1434 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [6.2320s] nsock_write(): Write request for 16 bytes to IOD #7 EID 1443 [10.10.0.201:23]
NSOCK INFO [6.2320s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1443 [10.10.0.201:23]
NSOCK INFO [6.2320s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 7000ms) EID 1450
NSOCK INFO [6.2450s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1450 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [6.2450s] nsock_write(): Write request for 3 bytes to IOD #7 EID 1459 [10.10.0.201:23]
NSOCK INFO [6.2450s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1459 [10.10.0.201:23]
NSOCK INFO [6.2450s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 7000ms) EID 1466
NSOCK INFO [6.2470s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1466 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [6.2470s] nsock_write(): Write request for 3 bytes to IOD #7 EID 1475 [10.10.0.201:23]
NSOCK INFO [6.2470s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1475 [10.10.0.201:23]
NSOCK INFO [6.2470s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 7000ms) EID 1482
NSOCK INFO [6.2860s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1482 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [6.2860s] nsock_write(): Write request for 3 bytes to IOD #7 EID 1491 [10.10.0.201:23]
NSOCK INFO [6.2860s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1491 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [6.2860s] nsock_write(): Write request for 3 bytes to IOD #7 EID 1499 [10.10.0.201:23]
NSOCK INFO [6.2860s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1499 [10.10.0.201:23]
NSOCK INFO [6.2870s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 7000ms) EID 1506
NSOCK INFO [6.5520s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1506 [10.10.0.201:23] (524 bytes)
NSOCK INFO [6.5530s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 7000ms) EID 1514
NSOCK INFO [6.5540s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1514 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]                                                                                W
NSE: [tso-brute W:1f65128 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute W:1f65128 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute W:1f65128 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute W:1f65128 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute W:1f65128 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute W:1f65128 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute W:1f65128 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute W:1f65128 10.10.0.201:23] fficials.
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying @AIC/fake against 10.10.0.201:23
NSE: [tso-brute W:1f65128 10.10.0.201:23] Getting to TSO
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AIC/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying User ID: @AIC
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7d50d91150d1a3a296407cc1c9c3ffef
NSOCK INFO [6.5620s] nsock_write(): Write request for 16 bytes to IOD #7 EID 1523 [10.10.0.201:23]
NSOCK INFO [6.5620s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1523 [10.10.0.201:23]
NSOCK INFO [6.5630s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1530
NSOCK INFO [6.6000s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1530 [10.10.0.201:23] (524 bytes)
NSOCK INFO [6.6010s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1538
NSOCK INFO [6.6010s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1538 [10.10.0.201:23] (122 bytes)
NSOCK INFO [6.6110s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1546
NSOCK INFO [6.6680s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1546 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [6.6690s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1554
NSOCK INFO [6.6890s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1554 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [6.6900s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1562
NSOCK INFO [6.6920s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1562 [10.10.0.201:23] (524 bytes)
NSOCK INFO [6.6930s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1570
NSOCK INFO [6.6930s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1570 [10.10.0.201:23] (582 bytes)
NSOCK INFO [6.7070s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1578
NSOCK INFO [6.9070s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 1578 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User ID: @AIC
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AIC                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] @AIC is a valid TSO User ID. Trying Password: fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7dc9c711c9c386819285ffef
NSOCK INFO [6.9220s] nsock_write(): Write request for 12 bytes to IOD #7 EID 1587 [10.10.0.201:23]
NSOCK INFO [6.9220s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1587 [10.10.0.201:23]
NSOCK INFO [6.9220s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1594
NSOCK INFO [6.9430s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1594 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [6.9440s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1602
NSOCK INFO [6.9510s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1602 [10.10.0.201:23] (8 bytes): ...@@...
NSOCK INFO [6.9520s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1610
NSOCK INFO [6.9940s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1610 [10.10.0.201:23] (524 bytes)
NSOCK INFO [6.9950s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1618
NSOCK INFO [6.9950s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1618 [10.10.0.201:23] (582 bytes)
NSOCK INFO [7.0050s] nsock_read(): Read request from IOD #7 [10.10.0.201:23] (timeout: 200ms) EID 1626
NSOCK INFO [7.2060s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 1626 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User/Pass: @AIC/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56421I PASSWORD NOT AUTHORIZED FOR USERID
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56429A REENTER -
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AIC                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Disconnecting
NSOCK INFO [7.2060s] nsock_iod_delete(): nsock_iod_delete (IOD #7)
NSOCK INFO [7.2060s] nsock_iod_new2(): nsock_iod_new (IOD #8)
NSOCK INFO [7.2170s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #8) EID 1632
NSOCK INFO [7.2280s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 1632 [10.10.0.201:23]
NSOCK INFO [7.2280s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 7000ms) EID 1642
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Status: #threads = 1, #retry_accounts = 0, initial_accounts_exhausted = false, waiting = 0
NSOCK INFO [7.2480s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1642 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [7.2480s] nsock_write(): Write request for 3 bytes to IOD #8 EID 1651 [10.10.0.201:23]
NSOCK INFO [7.2480s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1651 [10.10.0.201:23]
NSOCK INFO [7.2480s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 7000ms) EID 1658
NSOCK INFO [7.2590s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1658 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [7.2590s] nsock_write(): Write request for 3 bytes to IOD #8 EID 1667 [10.10.0.201:23]
NSOCK INFO [7.2590s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1667 [10.10.0.201:23]
NSOCK INFO [7.2590s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 7000ms) EID 1674
NSOCK INFO [7.2670s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1674 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [7.2670s] nsock_write(): Write request for 16 bytes to IOD #8 EID 1683 [10.10.0.201:23]
NSOCK INFO [7.2670s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1683 [10.10.0.201:23]
NSOCK INFO [7.2670s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 7000ms) EID 1690
NSOCK INFO [7.2800s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1690 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [7.2800s] nsock_write(): Write request for 3 bytes to IOD #8 EID 1699 [10.10.0.201:23]
NSOCK INFO [7.2800s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1699 [10.10.0.201:23]
NSOCK INFO [7.2800s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 7000ms) EID 1706
NSOCK INFO [7.2890s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1706 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [7.2890s] nsock_write(): Write request for 3 bytes to IOD #8 EID 1715 [10.10.0.201:23]
NSOCK INFO [7.2890s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1715 [10.10.0.201:23]
NSOCK INFO [7.2890s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 7000ms) EID 1722
NSOCK INFO [7.3300s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1722 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [7.3300s] nsock_write(): Write request for 3 bytes to IOD #8 EID 1731 [10.10.0.201:23]
NSOCK INFO [7.3300s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1731 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [7.3300s] nsock_write(): Write request for 3 bytes to IOD #8 EID 1739 [10.10.0.201:23]
NSOCK INFO [7.3300s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1739 [10.10.0.201:23]
NSOCK INFO [7.3310s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 7000ms) EID 1746
NSOCK INFO [7.5960s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1746 [10.10.0.201:23] (524 bytes)
NSOCK INFO [7.5970s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 7000ms) EID 1754
NSOCK INFO [7.5970s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1754 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]                                                                                W
NSE: [tso-brute W:1f65128 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute W:1f65128 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute W:1f65128 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute W:1f65128 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute W:1f65128 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute W:1f65128 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute W:1f65128 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute W:1f65128 10.10.0.201:23] fficials.
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying @AGA/fake against 10.10.0.201:23
NSE: [tso-brute W:1f65128 10.10.0.201:23] Getting to TSO
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AGA/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying User ID: @AGA
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7d50d91150d1a3a296407cc1c7c1ffef
NSOCK INFO [7.6080s] nsock_write(): Write request for 16 bytes to IOD #8 EID 1763 [10.10.0.201:23]
NSOCK INFO [7.6080s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1763 [10.10.0.201:23]
NSOCK INFO [7.6080s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1770
NSOCK INFO [7.6410s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1770 [10.10.0.201:23] (524 bytes)
NSOCK INFO [7.6420s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1778
NSOCK INFO [7.6420s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1778 [10.10.0.201:23] (122 bytes)
NSOCK INFO [7.6500s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1786
NSOCK INFO [7.6980s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1786 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [7.6990s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1794
NSOCK INFO [7.7220s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1794 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [7.7220s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1802
NSOCK INFO [7.7250s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1802 [10.10.0.201:23] (524 bytes)
NSOCK INFO [7.7260s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1810
NSOCK INFO [7.7260s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1810 [10.10.0.201:23] (582 bytes)
NSOCK INFO [7.7410s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1818
NSOCK INFO [7.9410s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 1818 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User ID: @AGA
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AGA                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] @AGA is a valid TSO User ID. Trying Password: fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7dc9c711c9c386819285ffef
NSOCK INFO [7.9560s] nsock_write(): Write request for 12 bytes to IOD #8 EID 1827 [10.10.0.201:23]
NSOCK INFO [7.9560s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1827 [10.10.0.201:23]
NSOCK INFO [7.9560s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1834
NSOCK INFO [7.9710s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1834 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [7.9720s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1842
NSOCK INFO [7.9760s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1842 [10.10.0.201:23] (8 bytes): ...@@...
NSOCK INFO [7.9760s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1850
NSOCK INFO [8.0040s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1850 [10.10.0.201:23] (524 bytes)
NSOCK INFO [8.0050s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1858
NSOCK INFO [8.0050s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1858 [10.10.0.201:23] (524 bytes)
NSOCK INFO [8.0060s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1866
NSOCK INFO [8.0060s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1866 [10.10.0.201:23] (58 bytes): @@@@@@@@@@@@......|@........@@@@@~~~n....|@@@@@@@@........
NSOCK INFO [8.0160s] nsock_read(): Read request from IOD #8 [10.10.0.201:23] (timeout: 200ms) EID 1874
NSOCK INFO [8.2170s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 1874 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User/Pass: @AGA/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56421I PASSWORD NOT AUTHORIZED FOR USERID
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56429A REENTER -
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AGA                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Disconnecting
NSOCK INFO [8.2180s] nsock_iod_delete(): nsock_iod_delete (IOD #8)
NSOCK INFO [8.2180s] nsock_iod_new2(): nsock_iod_new (IOD #9)
NSOCK INFO [8.2290s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #9) EID 1880
NSOCK INFO [8.2340s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 1880 [10.10.0.201:23]
NSOCK INFO [8.2340s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 7000ms) EID 1890
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Status: #threads = 1, #retry_accounts = 0, initial_accounts_exhausted = false, waiting = 0
NSOCK INFO [8.2490s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1890 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [8.2490s] nsock_write(): Write request for 3 bytes to IOD #9 EID 1899 [10.10.0.201:23]
NSOCK INFO [8.2490s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1899 [10.10.0.201:23]
NSOCK INFO [8.2490s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 7000ms) EID 1906
NSOCK INFO [8.2560s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1906 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [8.2570s] nsock_write(): Write request for 3 bytes to IOD #9 EID 1915 [10.10.0.201:23]
NSOCK INFO [8.2570s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1915 [10.10.0.201:23]
NSOCK INFO [8.2570s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 7000ms) EID 1922
NSOCK INFO [8.2680s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1922 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [8.2690s] nsock_write(): Write request for 16 bytes to IOD #9 EID 1931 [10.10.0.201:23]
NSOCK INFO [8.2690s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1931 [10.10.0.201:23]
NSOCK INFO [8.2690s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 7000ms) EID 1938
NSOCK INFO [8.2920s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1938 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [8.2920s] nsock_write(): Write request for 3 bytes to IOD #9 EID 1947 [10.10.0.201:23]
NSOCK INFO [8.2920s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1947 [10.10.0.201:23]
NSOCK INFO [8.2920s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 7000ms) EID 1954
NSOCK INFO [8.2930s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1954 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [8.2940s] nsock_write(): Write request for 3 bytes to IOD #9 EID 1963 [10.10.0.201:23]
NSOCK INFO [8.2940s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1963 [10.10.0.201:23]
NSOCK INFO [8.2940s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 7000ms) EID 1970
NSOCK INFO [8.3340s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1970 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [8.3340s] nsock_write(): Write request for 3 bytes to IOD #9 EID 1979 [10.10.0.201:23]
NSOCK INFO [8.3340s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1979 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [8.3340s] nsock_write(): Write request for 3 bytes to IOD #9 EID 1987 [10.10.0.201:23]
NSOCK INFO [8.3340s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 1987 [10.10.0.201:23]
NSOCK INFO [8.3340s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 7000ms) EID 1994
NSOCK INFO [8.6020s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 1994 [10.10.0.201:23] (524 bytes)
NSOCK INFO [8.6030s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 7000ms) EID 2002
NSOCK INFO [8.6030s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2002 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]                                                                                W
NSE: [tso-brute W:1f65128 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute W:1f65128 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute W:1f65128 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute W:1f65128 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute W:1f65128 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute W:1f65128 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute W:1f65128 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute W:1f65128 10.10.0.201:23] fficials.
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying @AHE/fake against 10.10.0.201:23
NSE: [tso-brute W:1f65128 10.10.0.201:23] Getting to TSO
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AHE/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying User ID: @AHE
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7d50d91150d1a3a296407cc1c8c5ffef
NSOCK INFO [8.6140s] nsock_write(): Write request for 16 bytes to IOD #9 EID 2011 [10.10.0.201:23]
NSOCK INFO [8.6150s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2011 [10.10.0.201:23]
NSOCK INFO [8.6150s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2018
NSOCK INFO [8.6480s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2018 [10.10.0.201:23] (524 bytes)
NSOCK INFO [8.6500s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2026
NSOCK INFO [8.6500s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2026 [10.10.0.201:23] (122 bytes)
NSOCK INFO [8.6560s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2034
NSOCK INFO [8.7020s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2034 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [8.7020s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2042
NSOCK INFO [8.7220s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2042 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [8.7230s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2050
NSOCK INFO [8.7270s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2050 [10.10.0.201:23] (524 bytes)
NSOCK INFO [8.7280s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2058
NSOCK INFO [8.7280s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2058 [10.10.0.201:23] (524 bytes)
NSOCK INFO [8.7300s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2066
NSOCK INFO [8.7300s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2066 [10.10.0.201:23] (58 bytes): @@@@@@@@@@@@......|@........@@@@@~~~n....|@@@@@@@@........
NSOCK INFO [8.7420s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2074
NSOCK INFO [8.9420s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 2074 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User ID: @AHE
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AHE                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] @AHE is a valid TSO User ID. Trying Password: fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7dc9c711c9c386819285ffef
NSOCK INFO [8.9560s] nsock_write(): Write request for 12 bytes to IOD #9 EID 2083 [10.10.0.201:23]
NSOCK INFO [8.9560s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2083 [10.10.0.201:23]
NSOCK INFO [8.9560s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2090
NSOCK INFO [8.9770s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2090 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [8.9770s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2098
NSOCK INFO [8.9840s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2098 [10.10.0.201:23] (8 bytes): ...@@...
NSOCK INFO [8.9840s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2106
NSOCK INFO [9.0100s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2106 [10.10.0.201:23] (524 bytes)
NSOCK INFO [9.0110s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2114
NSOCK INFO [9.0110s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2114 [10.10.0.201:23] (582 bytes)
NSOCK INFO [9.0220s] nsock_read(): Read request from IOD #9 [10.10.0.201:23] (timeout: 200ms) EID 2122
NSOCK INFO [9.2230s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 2122 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User/Pass: @AHE/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56421I PASSWORD NOT AUTHORIZED FOR USERID
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56429A REENTER -
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AHE                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Disconnecting
NSOCK INFO [9.2240s] nsock_iod_delete(): nsock_iod_delete (IOD #9)
NSOCK INFO [9.2240s] nsock_iod_new2(): nsock_iod_new (IOD #10)
NSOCK INFO [9.2340s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #10) EID 2128
NSOCK INFO [9.2410s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 2128 [10.10.0.201:23]
NSOCK INFO [9.2410s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 7000ms) EID 2138
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Status: #threads = 1, #retry_accounts = 0, initial_accounts_exhausted = false, waiting = 0
NSOCK INFO [9.2550s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2138 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [9.2550s] nsock_write(): Write request for 3 bytes to IOD #10 EID 2147 [10.10.0.201:23]
NSOCK INFO [9.2550s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2147 [10.10.0.201:23]
NSOCK INFO [9.2550s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 7000ms) EID 2154
NSOCK INFO [9.2650s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2154 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [9.2650s] nsock_write(): Write request for 3 bytes to IOD #10 EID 2163 [10.10.0.201:23]
NSOCK INFO [9.2650s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2163 [10.10.0.201:23]
NSOCK INFO [9.2650s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 7000ms) EID 2170
NSOCK INFO [9.2700s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2170 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [9.2710s] nsock_write(): Write request for 16 bytes to IOD #10 EID 2179 [10.10.0.201:23]
NSOCK INFO [9.2710s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2179 [10.10.0.201:23]
NSOCK INFO [9.2710s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 7000ms) EID 2186
NSOCK INFO [9.2760s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2186 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [9.2760s] nsock_write(): Write request for 3 bytes to IOD #10 EID 2195 [10.10.0.201:23]
NSOCK INFO [9.2760s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2195 [10.10.0.201:23]
NSOCK INFO [9.2760s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 7000ms) EID 2202
NSOCK INFO [9.2790s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2202 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [9.2790s] nsock_write(): Write request for 3 bytes to IOD #10 EID 2211 [10.10.0.201:23]
NSOCK INFO [9.2790s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2211 [10.10.0.201:23]
NSOCK INFO [9.2790s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 7000ms) EID 2218
NSOCK INFO [9.3200s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2218 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [9.3200s] nsock_write(): Write request for 3 bytes to IOD #10 EID 2227 [10.10.0.201:23]
NSOCK INFO [9.3200s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2227 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [9.3200s] nsock_write(): Write request for 3 bytes to IOD #10 EID 2235 [10.10.0.201:23]
NSOCK INFO [9.3200s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2235 [10.10.0.201:23]
NSOCK INFO [9.3210s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 7000ms) EID 2242
NSOCK INFO [9.5830s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2242 [10.10.0.201:23] (524 bytes)
NSOCK INFO [9.5850s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 7000ms) EID 2250
NSOCK INFO [9.5860s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2250 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]                                                                                W
NSE: [tso-brute W:1f65128 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute W:1f65128 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute W:1f65128 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute W:1f65128 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute W:1f65128 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute W:1f65128 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute W:1f65128 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute W:1f65128 10.10.0.201:23] fficials.
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying @AIF/fake against 10.10.0.201:23
NSE: [tso-brute W:1f65128 10.10.0.201:23] Getting to TSO
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AIF/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying User ID: @AIF
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7d50d91150d1a3a296407cc1c9c6ffef
NSOCK INFO [9.5960s] nsock_write(): Write request for 16 bytes to IOD #10 EID 2259 [10.10.0.201:23]
NSOCK INFO [9.5970s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2259 [10.10.0.201:23]
NSOCK INFO [9.5970s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2266
NSOCK INFO [9.6400s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2266 [10.10.0.201:23] (524 bytes)
NSOCK INFO [9.6410s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2274
NSOCK INFO [9.6410s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2274 [10.10.0.201:23] (122 bytes)
NSOCK INFO [9.6490s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2282
NSOCK INFO [9.7000s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2282 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [9.7010s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2290
NSOCK INFO [9.7270s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2290 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [9.7280s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2298
NSOCK INFO [9.7280s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2298 [10.10.0.201:23] (524 bytes)
NSOCK INFO [9.7290s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2306
NSOCK INFO [9.7290s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2306 [10.10.0.201:23] (582 bytes)
NSOCK INFO [9.7450s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2314
NSOCK INFO [9.9450s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 2314 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User ID: @AIF
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AIF                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] @AIF is a valid TSO User ID. Trying Password: fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7dc9c711c9c386819285ffef
NSOCK INFO [9.9590s] nsock_write(): Write request for 12 bytes to IOD #10 EID 2323 [10.10.0.201:23]
NSOCK INFO [9.9590s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2323 [10.10.0.201:23]
NSOCK INFO [9.9590s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2330
NSOCK INFO [9.9770s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2330 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [9.9770s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2338
NSOCK INFO [9.9810s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2338 [10.10.0.201:23] (8 bytes): ...@@...
NSOCK INFO [9.9820s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2346
NSOCK INFO [10.0140s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2346 [10.10.0.201:23] (524 bytes)
NSOCK INFO [10.0150s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2354
NSOCK INFO [10.0150s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2354 [10.10.0.201:23] (582 bytes)
NSOCK INFO [10.0270s] nsock_read(): Read request from IOD #10 [10.10.0.201:23] (timeout: 200ms) EID 2362
NSOCK INFO [10.2280s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 2362 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User/Pass: @AIF/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56421I PASSWORD NOT AUTHORIZED FOR USERID
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56429A REENTER -
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AIF                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Disconnecting
NSOCK INFO [10.2280s] nsock_iod_delete(): nsock_iod_delete (IOD #10)
NSOCK INFO [10.2280s] nsock_iod_new2(): nsock_iod_new (IOD #11)
NSOCK INFO [10.2380s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #11) EID 2368
NSOCK INFO [10.2430s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 2368 [10.10.0.201:23]
NSOCK INFO [10.2430s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 7000ms) EID 2378
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Status: #threads = 1, #retry_accounts = 0, initial_accounts_exhausted = false, waiting = 0
NSOCK INFO [10.2540s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2378 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [10.2540s] nsock_write(): Write request for 3 bytes to IOD #11 EID 2387 [10.10.0.201:23]
NSOCK INFO [10.2540s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2387 [10.10.0.201:23]
NSOCK INFO [10.2550s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 7000ms) EID 2394
NSOCK INFO [10.2610s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2394 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [10.2610s] nsock_write(): Write request for 3 bytes to IOD #11 EID 2403 [10.10.0.201:23]
NSOCK INFO [10.2610s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2403 [10.10.0.201:23]
NSOCK INFO [10.2610s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 7000ms) EID 2410
NSOCK INFO [10.2650s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2410 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [10.2650s] nsock_write(): Write request for 16 bytes to IOD #11 EID 2419 [10.10.0.201:23]
NSOCK INFO [10.2650s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2419 [10.10.0.201:23]
NSOCK INFO [10.2650s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 7000ms) EID 2426
NSOCK INFO [10.2730s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2426 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [10.2730s] nsock_write(): Write request for 3 bytes to IOD #11 EID 2435 [10.10.0.201:23]
NSOCK INFO [10.2730s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2435 [10.10.0.201:23]
NSOCK INFO [10.2730s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 7000ms) EID 2442
NSOCK INFO [10.2740s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2442 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [10.2740s] nsock_write(): Write request for 3 bytes to IOD #11 EID 2451 [10.10.0.201:23]
NSOCK INFO [10.2740s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2451 [10.10.0.201:23]
NSOCK INFO [10.2740s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 7000ms) EID 2458
NSOCK INFO [10.3150s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2458 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [10.3160s] nsock_write(): Write request for 3 bytes to IOD #11 EID 2467 [10.10.0.201:23]
NSOCK INFO [10.3160s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2467 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [10.3160s] nsock_write(): Write request for 3 bytes to IOD #11 EID 2475 [10.10.0.201:23]
NSOCK INFO [10.3160s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2475 [10.10.0.201:23]
NSOCK INFO [10.3160s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 7000ms) EID 2482
NSOCK INFO [10.5790s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2482 [10.10.0.201:23] (524 bytes)
NSOCK INFO [10.5800s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 7000ms) EID 2490
NSOCK INFO [10.5800s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2490 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]                                                                                W
NSE: [tso-brute W:1f65128 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute W:1f65128 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute W:1f65128 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute W:1f65128 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute W:1f65128 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute W:1f65128 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute W:1f65128 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute W:1f65128 10.10.0.201:23] fficials.
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying @AGC/fake against 10.10.0.201:23
NSE: [tso-brute W:1f65128 10.10.0.201:23] Getting to TSO
NSE: [tso-brute 10.10.0.201:23] Trying User ID/Password: @AGC/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Issuing Command (#1 of 1): tso
NSE: [tso-brute W:1f65128 10.10.0.201:23] Trying User ID: @AGC
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7d50d91150d1a3a296407cc1c7c3ffef
NSOCK INFO [10.5920s] nsock_write(): Write request for 16 bytes to IOD #11 EID 2499 [10.10.0.201:23]
NSOCK INFO [10.5920s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2499 [10.10.0.201:23]
NSOCK INFO [10.5920s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2506
NSOCK INFO [10.6270s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2506 [10.10.0.201:23] (524 bytes)
NSOCK INFO [10.6290s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2514
NSOCK INFO [10.6290s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2514 [10.10.0.201:23] (122 bytes)
NSOCK INFO [10.6380s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2522
NSOCK INFO [10.6950s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2522 [10.10.0.201:23] (10 bytes): ...@@.@...
NSOCK INFO [10.6960s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2530
NSOCK INFO [10.7140s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2530 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [10.7150s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2538
NSOCK INFO [10.7170s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2538 [10.10.0.201:23] (524 bytes)
NSOCK INFO [10.7190s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2546
NSOCK INFO [10.7190s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2546 [10.10.0.201:23] (524 bytes)
NSOCK INFO [10.7210s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2554
NSOCK INFO [10.7210s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2554 [10.10.0.201:23] (58 bytes): @@@@@@@@@@@@......|@........@@@@@~~~n....|@@@@@@@@........
NSOCK INFO [10.7330s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2562
NSOCK INFO [10.9330s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 2562 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User ID: @AGC
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AGC                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] @AGC is a valid TSO User ID. Trying Password: fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0x7dc9c711c9c386819285ffef
NSOCK INFO [10.9470s] nsock_write(): Write request for 12 bytes to IOD #11 EID 2571 [10.10.0.201:23]
NSOCK INFO [10.9470s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2571 [10.10.0.201:23]
NSOCK INFO [10.9470s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2578
NSOCK INFO [10.9660s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2578 [10.10.0.201:23] (13 bytes): ...]..@.@@...
NSOCK INFO [10.9670s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2586
NSOCK INFO [10.9720s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2586 [10.10.0.201:23] (8 bytes): ...@@...
NSOCK INFO [10.9720s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2594
NSOCK INFO [11.0070s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2594 [10.10.0.201:23] (524 bytes)
NSOCK INFO [11.0080s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2602
NSOCK INFO [11.0080s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2602 [10.10.0.201:23] (582 bytes)
NSOCK INFO [11.0190s] nsock_read(): Read request from IOD #11 [10.10.0.201:23] (timeout: 200ms) EID 2610
NSOCK INFO [11.2200s] nsock_trace_handler_callback(): Callback: READ TIMEOUT for EID 2610 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Screen Recieved for User/Pass: @AGC/fake
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] ------------------------------- TSO/E LOGON ------------------------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56421I PASSWORD NOT AUTHORIZED FOR USERID
NSE: [tso-brute W:1f65128 10.10.0.201:23] IKJ56429A REENTER -
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter LOGON parameters below:                   RACF LOGON parameters:
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Userid    ===> @AGC                             Seclabel     ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Password  ===>                                  New Password ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Procedure ===> ISPFPROC                         Group Ident  ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Acct Nmbr ===> ACCT#
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Size      ===> 4048
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Perform   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Command   ===>
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]    Enter an 'S' before each option desired below:
NSE: [tso-brute W:1f65128 10.10.0.201:23]            -Nomail         -Nonotice        -Reconnect        -OIDcard
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] PF1/PF13 ==> Help    PF3/PF15 ==> Logoff    PA1 ==> Attention    PA2 ==> Reshow
NSE: [tso-brute W:1f65128 10.10.0.201:23] You may request specific help information by entering a '?' in any entry field
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Disconnecting
NSOCK INFO [11.2200s] nsock_iod_delete(): nsock_iod_delete (IOD #11)
NSOCK INFO [11.2200s] nsock_iod_new2(): nsock_iod_new (IOD #12)
NSOCK INFO [11.2300s] nsock_connect_tcp(): TCP connection requested to 10.10.0.201:23 (IOD #12) EID 2616
NSOCK INFO [11.2380s] nsock_trace_handler_callback(): Callback: CONNECT SUCCESS for EID 2616 [10.10.0.201:23]
NSOCK INFO [11.2380s] nsock_read(): Read request from IOD #12 [10.10.0.201:23] (timeout: 7000ms) EID 2626
NSE: [tso-brute M:2d918f8 10.10.0.201:23] Status: #threads = 1, #retry_accounts = 0, initial_accounts_exhausted = false, waiting = 0
NSOCK INFO [11.2620s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2626 [10.10.0.201:23] (3 bytes): ..(
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffc28
NSOCK INFO [11.2630s] nsock_write(): Write request for 3 bytes to IOD #12 EID 2635 [10.10.0.201:23]
NSOCK INFO [11.2630s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2635 [10.10.0.201:23]
NSOCK INFO [11.2630s] nsock_read(): Read request from IOD #12 [10.10.0.201:23] (timeout: 7000ms) EID 2642
NSOCK INFO [11.2690s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2642 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb18
NSOCK INFO [11.2690s] nsock_write(): Write request for 3 bytes to IOD #12 EID 2651 [10.10.0.201:23]
NSOCK INFO [11.2690s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2651 [10.10.0.201:23]
NSOCK INFO [11.2690s] nsock_read(): Read request from IOD #12 [10.10.0.201:23] (timeout: 7000ms) EID 2658
NSOCK INFO [11.2750s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2658 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffa180049424d2d333237382d32fff0
NSOCK INFO [11.2750s] nsock_write(): Write request for 16 bytes to IOD #12 EID 2667 [10.10.0.201:23]
NSOCK INFO [11.2750s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2667 [10.10.0.201:23]
NSOCK INFO [11.2750s] nsock_read(): Read request from IOD #12 [10.10.0.201:23] (timeout: 7000ms) EID 2674
NSOCK INFO [11.2850s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2674 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb19
NSOCK INFO [11.2850s] nsock_write(): Write request for 3 bytes to IOD #12 EID 2683 [10.10.0.201:23]
NSOCK INFO [11.2850s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2683 [10.10.0.201:23]
NSOCK INFO [11.2860s] nsock_read(): Read request from IOD #12 [10.10.0.201:23] (timeout: 7000ms) EID 2690
NSOCK INFO [11.2860s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2690 [10.10.0.201:23] (3 bytes): ...
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd19
NSOCK INFO [11.2860s] nsock_write(): Write request for 3 bytes to IOD #12 EID 2699 [10.10.0.201:23]
NSOCK INFO [11.2860s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2699 [10.10.0.201:23]
NSOCK INFO [11.2860s] nsock_read(): Read request from IOD #12 [10.10.0.201:23] (timeout: 7000ms) EID 2706
NSOCK INFO [11.3280s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2706 [10.10.0.201:23] (6 bytes): ......
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffb00
NSOCK INFO [11.3280s] nsock_write(): Write request for 3 bytes to IOD #12 EID 2715 [10.10.0.201:23]
NSOCK INFO [11.3280s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2715 [10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] Sending data: 0xfffd00
NSOCK INFO [11.3280s] nsock_write(): Write request for 3 bytes to IOD #12 EID 2723 [10.10.0.201:23]
NSOCK INFO [11.3280s] nsock_trace_handler_callback(): Callback: WRITE SUCCESS for EID 2723 [10.10.0.201:23]
NSOCK INFO [11.3290s] nsock_read(): Read request from IOD #12 [10.10.0.201:23] (timeout: 7000ms) EID 2730
NSOCK INFO [11.5710s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2730 [10.10.0.201:23] (524 bytes)
NSOCK INFO [11.5730s] nsock_read(): Read request from IOD #12 [10.10.0.201:23] (timeout: 7000ms) EID 2738
NSOCK INFO [11.5730s] nsock_trace_handler_callback(): Callback: READ SUCCESS for EID 2738 [10.10.0.201:23] (122 bytes)
NSE: [tso-brute W:1f65128 10.10.0.201:23] ---------------------- Printing the current TN3270 buffer ----------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23]  Welcome to the E Corp Network
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]                                                                                W
NSE: [tso-brute W:1f65128 10.10.0.201:23] ARNING. You have accessed a private computer system. Unauthorized access, use, c
NSE: [tso-brute W:1f65128 10.10.0.201:23] onnection, or entry is not permitted and constitutes a crime punishable by law.W
NSE: [tso-brute W:1f65128 10.10.0.201:23] e reserve the right to fully pursue criminal and civil legal penalties. All    i
NSE: [tso-brute W:1f65128 10.10.0.201:23] ndividuals using this computer system with or without proper authority are     s
NSE: [tso-brute W:1f65128 10.10.0.201:23] ubject to having all their activities monitored and recorded. Anyone using     t
NSE: [tso-brute W:1f65128 10.10.0.201:23] his system implicitly consents to this monitoring. Any evidence of suspected   c
NSE: [tso-brute W:1f65128 10.10.0.201:23] riminal activity revealed by such monitoring may be provided to law enforcemento
NSE: [tso-brute W:1f65128 10.10.0.201:23] fficials.
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23]
NSE: [tso-brute W:1f65128 10.10.0.201:23] ----------------------- End of the current TN3270 buffer ---------------------
NSE: [tso-brute W:1f65128 10.10.0.201:23] Disconnecting
NSOCK INFO [11.5730s] nsock_iod_delete(): nsock_iod_delete (IOD #12)
NSE: Finished tso-brute W:1f65128 against 10.10.0.201:23.
NSE: Finished tso-brute M:2d918f8 against 10.10.0.201:23.
Completed NSE at 09:56, 11.11s elapsed
NSE: Starting runlevel 2 (of 2) scan.
Initiating NSE at 09:56
Completed NSE at 09:56, 0.00s elapsed
Nmap scan report for 10.10.0.201
Host is up, received syn-ack (0.0078s latency).
Scanned at 2017-01-19 09:56:48 PST for 11s
PORT   STATE SERVICE REASON  VERSION
23/tcp open  tn3270  syn-ack IBM Telnet TN3270 (TN3270E)
| tso-brute:
|   Accounts: No valid accounts found
|_  Statistics: Performed 9 guesses in 9 seconds, average tps: 1.0
Final times for host: srtt: 7779 rttvar: 7143  to: 100000

NSE: Script Post-scanning.
NSE: Starting runlevel 1 (of 2) scan.
Initiating NSE at 09:56
Completed NSE at 09:56, 0.00s elapsed
NSE: Starting runlevel 2 (of 2) scan.
Initiating NSE at 09:56
Completed NSE at 09:56, 0.00s elapsed
Read from /usr/local/bin/../share/nmap: nmap-payloads nmap-service-probes nmap-services.
Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .
Nmap done: 1 IP address (1 host up) scanned in 11.59 seconds
```